WordPress/twentytwentyone
# Update password input field styles

Fixes https://github.com/WordPress/twentytwentyone/issues/813

## Summary

Moved the password form styles to forms.scss.

Fixed a problem where the generic style for the password input field
was overriding the margins for the password protected posts.

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1.Enable password protection for a post
1. View that post on the front, to see the password form.
1. See that the height of the field and button match.


## Screenshots

After: 
![password-after](https://user-images.githubusercontent.com/7422055/98534454-c9b45c00-2284-11eb-973b-1b5bdb7f95b0.png)


## Quality assurance
